src/index.ts

```diff
@@ -221,7 +221,7 @@ prog
       process.chdir(projectPath)
       const safeName = safePackageName(pkg)
       // eslint-disable-next-line sort-keys
-      const pkgJson = generatePackageJson({ name: safeName, author })
+      const pkgJson = generatePackageJson({ author, name: safeName })
```

Comment:
Isn't there a type for `author`?

Reply:
I'm assuming it should be string but this was an auto-formatted change so I'm weary of making changes I don't understand.

Reply:
Type added in dd90c30888cf30ba57a2dc92c492cc7465268a5e